trace/backend_collector.go

```diff
@@ -371,9 +376,9 @@ func (c *BackendCollector) queueUpload(w *witnessWithInfo) {
 		excludeWitnessFromReproMode(w.witness) {
 		// Obfuscate the original value so type inference engine can use it on the
 		// backend without revealing the actual value.
-		c.obfuscator.ZeroAllPrimitivesInMethod(w.witness.GetMethod())
+		data_masks.ObfuscateMethod(w.witness.GetMethod())
```

Comment:
Non-blocking: why bring back 'Obfuscate' over the more specific 'ZeroAllPrimitives'?